app/templates/components/popup-modal.hbs

```diff
@@ -14,9 +17,12 @@
 
         {{#each popup-btns as |btn|}}
         {{#link-to path class="popup-link" invokeAction=(if (not-eq btn.actionName 'null') (action (action 'buttonAction' btn.actionName))) 
```

Comment:
This checks for the`btn.actionName` to be with the string `'null'` Is this expected ? We can directly write condition as `(if btn.actionName ...)`.

Reply:
Its done.